src/config/HealthConfig.ts

```diff
@@ -28,8 +29,13 @@ export const getExpr = (value: RegexConfig | undefined): RegExp => {
   return allMatch;
 };
 
+const replaceXCode = (value: string): string => {
```

Comment:
Please, just add a minor comment to help with the context something like

"4XX and 5XX expressions are allowed in the code field"